tools/container/run-matrix.sh

```
@@ -0,0 +1,121 @@
+
+    if [[ -z "${name}" ]]; then
+        echo "[container] ERROR: matrix entry missing name" >&2
+        return 1
+    fi
+
+    if [[ "${TARGET}" == "tidy" && "${run_tidy}" != "1" ]]; then
+        echo "[container:${name}] Skipping target ${TARGET} (RUN_TIDY=0)"
+        return 2
+    fi
+
+    local docker_cmd="/workspace/tools/container/invoke.sh $(printf '%q' "${TARGET}")"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)